tests/conftest.py

```diff
@@ -1877,6 +1877,19 @@ def __dut_reload(duts_data, node=None, results=None):
     config_reload(node, wait_before_force_reload=300)
 
 
+def compare_running_config(pre_running_config, cur_running_config):
+    if pre_running_config == cur_running_config:
+        return True
+    if type(pre_running_config) != type(cur_running_config):
+        return False
+    if sorted(pre_running_config.keys()) != sorted(cur_running_config.keys()):
```

Comment:
How about using set to do the key comparison? Then no need to sort the list.

Reply:
Good idea~